src/test/java/com/logistic/project/controller/LogisticControllerTest.java

```diff
@@ -0,0 +1,81 @@
+package com.logistic.project.controller;
+
+import org.junit.ClassRule;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Testcontainers
+class LogisticControllerTest {
+    @ClassRule
+    private DockerComposeContainer compose = new DockerComposeContainer(
+            new File("test/resources/dockerCompose.yml"));
```

Comment:
Please try to rewrite it without using test/resources

Reply:
still problem with configuring classpath